cmd/sifnoded/cmd/add_genesis_validator_test.go

```diff
@@ -0,0 +1,66 @@
+package cmd_test
+
+import (
+	"bytes"
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/cosmos/cosmos-sdk/server"
+	svrcmd "github.com/cosmos/cosmos-sdk/server/cmd"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	genutiltypes "github.com/cosmos/cosmos-sdk/x/genutil/types"
+	"github.com/stretchr/testify/require"
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+
+	"github.com/Sifchain/sifnode/app"
+	sifnodedcmd "github.com/Sifchain/sifnode/cmd/sifnoded/cmd"
+	"github.com/Sifchain/sifnode/x/oracle"
+)
+
+func TestAddGenesisValidatorCmd(t *testing.T) {
+	homeDir, err := ioutil.TempDir("", "")
+	require.NoError(t, err)
+	defer os.RemoveAll(homeDir)
+
+	initCmd, _ := sifnodedcmd.NewRootCmd()
+	initBuf := new(bytes.Buffer)
+	initCmd.SetOut(initBuf)
+	initCmd.SetErr(initBuf)
+	initCmd.SetArgs([]string{"init", "test", "--home=" + homeDir})
+
+	app.SetConfig(true)
+	expectedValidatorBech32 := "sifvaloper1rwqp4q88ue83ag3kgnmxxypq0td59df4782tjn"
+	expectedValidator, err := sdk.ValAddressFromBech32(expectedValidatorBech32)
+	require.NoError(t, err)
+
+	addValCmd, _ := sifnodedcmd.NewRootCmd()
+	addValBuf := new(bytes.Buffer)
+	addValCmd.SetOut(addValBuf)
```

Comment:
minor: FYI, these only set stdout and stderr for the commands themselves - it's basically just for cobra useage output, not command stdout and stderr.  I think these are no-ops in the test.